### DISCUSSION TITLE
Expose getDependencies() for SubscriberPackageVersion class

### DISCUSSION THREAD

**Discussion by ClayChipps** [forcedotcom/cli#2625] [Ideas]
Currently, the @salesforce/packaging [SubscriberPackageVersion class ](https://github.com/forcedotcom/packaging/blob/main/src/package/subscriberPackageVersion.ts) does not expose a public getter function to retrieve the dependencies.

I am proposing adding a new getter function `getDependencies()` that will allow a clean api to get the package dependencies. (Similar to `getRemoteSiteSettings()`).

**Comment by shetzel** [comment]
Hi Clay!  I see no reason to not add this.  Would you like to handle the PR and I'll make sure it doesn't get held up?

**Reply by ClayChipps** [reply]
@shetzel Opened a PR [here](https://github.com/forcedotcom/packaging/pull/480). Thank you for the quick response!

**Reply by shetzel** [reply]
Merged.  Thanks for the contribution @ClayChipps !

**Comment by shetzel** [comment]
Function will be added in the release candidate version of the CLI on January 10th, or available in tomorrow's nightly release.
